en/docs/tutorials/integration-tutorials/service-catalog-tutorial.md

```diff
@@ -0,0 +1,212 @@
+
+    <img src="{{base_path}}/assets/img/integrate/create_project/create-integration-project.png" width="700">
+
+    This will open the <b>New Integration Project</b> dialog box.
+
+3.  Enter `ServiceCatalogSample` as the project name and select the following check boxes to create the required 
+    modules.
+    -   **Create ESB Configs**
+    -   **Create Composite Exporter**
+
+4.  Click **Finish**.
+
+    You can see the projects listed in the **Project Explorer** as shown below:
+
+    <img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/project-explorer-service-catalog.PNG">
+
+!!! Note
+    Note that a **resources** folder is created in the ServiceCatalogSampleConfigs project. This folder will be used to hold the swagger and metadata YAML files. These YAML files will be uploaded to the service catalog in upcoming steps.   
+
+#### Creating artifacts
+
+Create **QueryDoctorEP** and **HealthcareAPI** by following the instructions given in [Create an Endpoint](../sending-a-simple-message-to-a-service/#create-an-endpoint) and [Create a REST API](../sending-a-simple-message-to-a-service/#create-a-rest-api) sections of **Sending a Simple Message to a Service** 
+tutorial.
+
+Once the **HealthcareAPI** is created, two new files with following names will be added to the metadata directory automatically.
+
+1.  HealthcareAPI_metadata.yaml
+2.  HealthcareAPI_swagger.yaml
+
+<img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/metadata-folder-service-catalog.PNG" width="400">
+
+!!! Note
+    When registering a service in the Service Catalog, we need to provide the OpenAPI definition and metadata of the service we are registering. When creating an API, Integration Studio automatically creates these files for the user.
+
+### Step 2: Configuring meta information
+
+Open the HealthcareAPI_metadata.yaml and do the following changes.
+
+1.  Change the **description** to explain what the API does. ( Ex API to fetch doctors of a given category )
+2.  Change serviceUrl from https to http. ( since HealthcareAPI is not secured )
+
+<img src="{{base_path}}/assets/img/integrate/tutorials/service-catalog/edit-metadata-service-catalog.PNG">
+
+**serviceUrl** is the URL of the API when it gets deployed in the Micro Integrator. The API developer might not 
+know this information in the development time. So, the URL can be parametrized to be resolved later using environment 
+variables. 
+
+We can configure the serviceUrl in following ways.
+
+1.  Add the complete URL without parameters. Ex: http://localhost:8290/healthcare
+2.  Parameterize using the host and port combination. Ex: http://{host}:{port}/healthcare
+3.  Parameterize using a pre-configured URL. Ex: http://{url}/healthcare
+
+We will configure the environment variables to resolve {host} and {port} configurations in an upcoming step. 
+
+Please [refer](link) to get more information on the other fields in the metadata YAML file.
```

Comment:
The link is missing?

Reply:
We should link this to the document where the format (fields) of the metadata file is described.

Reply:
Linked an empty page for this as a temporary fix. We will be adding the REST API docs later. Fixed in a0c600c.